feat(issue-details): Update issue details docs for streamlined experience (#12480)
getsentry/sentry-docs: docs/product/issues/issue-details/index.mdx

description: "Learn how to navigate the Issue Details page to help you efficiently triage an issue."
---

> The **Issue Details** page helps you to gain further insight into the source of the issue and the impact it has on your application's users.

Reviewer: ```suggestion
The **Issue Details** page helps you gain further insight into the source of the issue and the impact it has on your application's users.
```

## Event Search

The embedded search bar allows you to search for events using any of the [event properties](/concepts/search/searchable-properties/events). Environment and time period can also be changed. Once the search is updated, the counts and graph below will be updated. Event navigations such as recommended, first, and last, and All Events will also be updated to only show matching events.

## Trends & Aggregates

The event and user counts represent the counts given the search, environments, or time period selected above. This is different from the counts in the header which are the total counts across the lifetime of the issue.

The graph is similarly affected by any search and can be toggled between user and event counts by the selection on the left. Releases and feature flag data can be hidden by clicking on the field in the upper right corner.

A few tags are shown as a preview on the page, but to see more, you can either click into any preview tag to see more details or click "All Tags" to see all tags for an issue. Tags are not affected by the search filters above.

## Sidebar

> The first and last time the issue has been seen will be dispayed. These values will depend on your environment selection and will updated if you change the environments. If you have releases set up, the first and last release the issue was seen in will be shown

Reviewer: ```suggestion
The first and last time the issue has been seen will be displayed here. These values will depend on your environment selection and will updated if you change the environments. If you have releases set up, the first and last release where the issue was seen will be shown.
```

Participants and viewers of an issue will be shown along with the time that users viewed the issue, which can be found if you hover over a user or see all viewers.

Similar and merged issues allow you to learn more about and control [grouped issues](/product/issues/grouping-and-fingerprints). Similar Issues can be used to find any other issues in your project that are similar to the one you are viewing. Merged issues shows any issues that have been merged into the issue you are viewing.

## Issue Content

In addition to events, information about replays, attachments, and feedback can be found from the page.

- **Replays** – A list of [replays](/product/explore/session-replay/) tied to error events included in the issue.
- **User Feedback** - Any feedback collected from users through the <PlatformLink to="/user-feedback/">SDK's user feedback</PlatformLink>.
- **<PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink>** - Any additional files stored, such as config, log, or screenshot files that are related to the error event.

> While attachments are affected by the search filters on the page, attachments and user feedback are not.

Reviewer: I'm having a tough time parsing the meaning here. Are attachments are affected by the search filters, or no?

Author: second one should say replays!

description: "Learn how to navigate the Issue Details page to help you efficiently triage an issue."
---

The **Issue Details** page helps you gain further insight into the source of the issue and the impact it has on your application's users.

![Issue Details breakdown](./img/issue-details-breakdown.png)

The header contains high-level information about the issue such as the error message, total counts of how often it has been seen, and how many users it has affected. Actions such as assigning, resolving, and archiving are also here.

Beneath the header is the event graph, showing a distribution of events and errors. You can use the search bar to filter which events are shown. Tags are shown and can be opened from here too.

> The sidebar shows more about the issue and includes information such as the last time and the first time the issue was see, if the issue is linked to any GitHub or Jira issues, issue activity and comments, participants and viewers, and Sentry AI features.

Reviewer 2: ```suggestion
The sidebar shows more about the issue and includes information such as the last time and the first time the issue was seen, if the issue is linked to any GitHub or Jira issues, issue activity and comments, participants and viewers, and Sentry AI features.
```

## Event Search

The embedded search bar allows you to search for events using any of the [event properties](/concepts/search/searchable-properties/events). Environment and time period can also be changed. Once the search is updated, the counts and graph below will be updated. Event navigations such as recommended, first, and last, and All Events will also be updated to only show matching events.

## Trends & Aggregates

The event and user counts represent the counts given the search, environments, or time period selected above. This is different from the counts in the header which are the total counts across the lifetime of the issue.

The graph is similarly affected by any search and can be toggled between user and event counts by the selection on the left. Releases and feature flag data can be hidden by clicking on the field in the upper right corner.

A few tags are shown as a preview on the page, but to see more, you can either click into any preview tag to see more details or click "All Tags" to see all tags for an issue. Tags are not affected by the search filters above.

## Sidebar

> The first and last time the issue has been seen will be dispayed here. These values will depend on your environment selection and will updated if you change the environments. If you have releases set up, the first and last release where the issue was seen in will be shown

Reviewer 2: ```suggestion
The first and last time the issue has been seen will be displayed here. These values will depend on your environment selection and will updated if you change the environments. If you have releases set up, the first and last release where the issue was seen in will be shown
```

> The first and last time the issue has been seen will be dispayed here. These values will depend on your environment selection and will updated if you change the environments. If you have releases set up, the first and last release where the issue was seen in will be shown

Reviewer 2: ```suggestion
The first and last time the issue has been seen will be dispayed here. These values will depend on your environment selection and will be updated if you change the environments. If you have releases set up, the first and last release where the issue was seen in will be shown
```

> The first and last time the issue has been seen will be dispayed here. These values will depend on your environment selection and will updated if you change the environments. If you have releases set up, the first and last release where the issue was seen in will be shown

Reviewer 2: ```suggestion
The first and last time the issue has been seen will be dispayed here. These values will depend on your environment selection and will updated if you change the environments. If you have releases set up, the first and last release where the issue was seen will be shown
```

To view other events, you can use the dropdown to skip to the latest or oldest event, or view the full list of events. You can change the default event in your [User Settings](https://sentry.io/settings/account/details/#defaultIssueEvent).

> In addition, you can scroll between events chronologically or view all events. All event navigations will respect any searchfilter on the page, including search queries, environments, or date filters.

Reviewer: ```suggestion
In addition, you can scroll between events chronologically or view all events. All event navigations will respect any search filter on the page, including search queries, environments, or date filters.
```

## Event Actions

> Relevant information about specific events can be found here including copy the link to the event or event ID. You can also view the JSON of the event here.

Reviewer: What do you mean by "including copy the link" here, should it be "a copy of the link"?

Author: updated it to hopefully read a bit more clearly

## Event Details

> Depending on the issue category and available event data, the Issue Details page displays some combination of the following sections.

Reviewer: ```suggestion
Depending on the issue category and available event data, the Issue Details page displays some combination of the following sections:
```

### Trace Preview

An abbreviated view of the related [trace](/product/sentry-basics/tracing/distributed-tracing) for the current transaction.

![Trace Related Issue](./img/trace_related_issue.png)

Sometimes the trace will be supplemented by a Trace-related Issue. A Trace-related Issue is a different issue that occurred during the same trace as the event in view. This is useful for understanding the context of the current issue and may help you identify its root cause. A Trace-related Issue allows the developer to quickly navigate between the two related events.

### Session Replay

If you’ve enabled [Session Replay](/product/explore/session-replay/), you’ll have access to a replay preview on the page (if there’s one associated with the event). Replays can be associated with both frontend and [backend errors](/product/explore/session-replay/getting-started#replays-for-backend-errors), as long as distributed tracing is set up. From here you can navigate to the rest of the replays recorded for this issue, or inspect this replay on the [Replay Details](/product/explore/session-replay/web/replay-details/) page.

### HTTP Request

> Information from the HTTP request incuding the query string, cookies, and headers.

Reviewer 2: ```suggestion
Information from the HTTP request including the query string, cookies, and headers.
```
